_help/specific-agencies/overview.md

```diff
@@ -1,6 +1,9 @@
 ---
+title: help.specific-agencies.overview
 order: 0
 redirect_from:
 - /help/specific-agencies/
 ---
+# {% t page.title %}
```

Comment:
This seems to work, but I'm kinda confused how `jekyll-multiple-languages-plugin` differentiates between `page.title` as a hash key path from `en.yml`, vs. as a resolution of `site.translations[site.lang].help['specific-agencies'].overview` 🤔

Reply:
Or... is it working? I only see one heading here:

https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.app.cloud.gov/preview/18f/identity-site/margolis-translate-agency-help-pages/es/help/specific-agencies/overview/

But two headings for the English version:

https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.app.cloud.gov/preview/18f/identity-site/margolis-translate-agency-help-pages/help/specific-agencies/overview/

Whichever is more correct, we should ideally not have two adjacent `<h1>` like in the second link.

Reply:
It's working but I forgot to remove the extra h1 from the content of the YAML, fixed in cc284b522d366bbf3e4c51fe13ab3ae111cf0b3e

Reply:
And the reason it works is because it's calling `t(page.title)` and the value of `page.title` happens to be a translation key string

Reply:
I also realized that "Overview" is a good navigation title but a bad page title, so I'm going to flip this one to not use the `page.title`

Reply:
a57c839227108674d89be27baf843efef4b3b049